manifest/parse.go

```diff
@@ -28,7 +28,13 @@ type metadata struct {
 }
 
 func (m metadata) String() string {
-	return fmt.Sprintf("%s, %s, %s (%s)", m.Metadata.Namespace, m.Metadata.Name, m.Kind, m.ApiVersion)
+	apiBase := m.ApiVersion
+	sp := strings.Split(apiBase, "/")
+	if len(sp) > 1 {
+		apiBase = strings.Join(sp[:len(sp)-1], "/")
+	}
+
+	return fmt.Sprintf("%s, %s, %s (%s)", m.Metadata.Namespace, m.Metadata.Name, m.Kind, apiBase)
```

Comment:
For anyone interested - This is the heuristic to treat both apiVersions `apps/v1` and `apps/v1beta` as just `apps`, so that we can [give the same manifest name in mapping results](https://github.com/databus23/helm-diff/blob/master/manifest/parse.go#L90-L99) for appropriate diffing.